### PR TITLE
fix: stop crashing from `objc_getClassList`

### PR DESCRIPTION
**Pull request by markemer**

### CODE REVIEW

File: ios/Capacitor/Capacitor/CapacitorBridge.swift [hunk 1]
```diff
@@ -5,6 +5,22 @@ import Cordova
 
 internal typealias CapacitorPlugin = CAPPlugin & CAPBridgedPlugin
 
+#warning("remove this before merging! To test for crash")
```
Comment (giralte-ionic) [inline ios/Capacitor/Capacitor/CapacitorBridge.swift:8]: I'm guessing you wanted to remove this?

**Review by giralte-ionic — CHANGES_REQUESTED** [review]
please also run a fat

File: ios/Capacitor/Capacitor/CapacitorBridge.swift [hunk 2]
```diff
@@ -277,30 +293,28 @@ open class CapacitorBridge: NSObject, CAPBridgeProtocol {
...
+                        }
                     }
                 }
+            } catch {
+                print(error)
+            }
+        }
+
+        for plugin in pluginList {
+            if let capPlugin = plugin as? CapacitorPlugin.Type {
```
Comment (Steven0351) [inline ios/Capacitor/Capacitor/CapacitorBridge.swift:315]: We still need to keep this check
```swift
if plugin is CAPInstancePlugin.Type { continue }
```

File: ios/Capacitor/Capacitor/CapacitorBridge.swift (hunk 1 above)
Comment (markemer) [reply ios/Capacitor/Capacitor/CapacitorBridge.swift]: Pulled it before I forget for real.

**Review by giralte-ionic — APPROVED** [review]

File: ios/Capacitor/Capacitor/CapacitorBridge.swift [hunk 3]
```diff
@@ -277,30 +281,32 @@ open class CapacitorBridge: NSObject, CAPBridgeProtocol {
...
+
+        for plugin in pluginList {
+            if plugin is CAPInstancePlugin.Type { continue }
+            if let capPlugin = plugin as? CapacitorPlugin.Type {
+                registerPlugin(capPlugin)
             }
-            classes.deallocate()
-        } else {
-            // register core plugins only
-            [CAPHttpPlugin.self, CAPConsolePlugin.self, CAPWebViewPlugin.self, CAPCookiesPlugin.self]
```
Comment (Steven0351) [inline ios/Capacitor/Capacitor/CapacitorBridge.swift:302]: It looks like we may not be registering the core plugins after this block has been removed.

**Review by theproducer — APPROVED** [review]

**Review by Steven0351 — APPROVED** [review]

**Pull request merged**